scripts/train_mallet.sh

```diff
@@ -0,0 +1,26 @@
+CORPUS=$1
```

Comment:
this is called by new dataset but can also be used to train a new topic model for an existing dataset, if you want to change the number of topics.